tests/spec/core/markdown-spec.js

```diff
@@ -41,69 +45,86 @@ describe("Core - Markdown", function() {
     };
     ops.config.format = "markdown";
     makeRSDoc(ops, function(doc) {
-      expect($(".note em", doc).length).toEqual(1);
-      expect($(".issue em", doc).length).toEqual(1);
-      expect($(".req em", doc).length).toEqual(1);
-      expect($(".req h3", doc).length).toEqual(1);
+      expect(doc.querySelector(".note em")).toBeTruthy();
+      expect(doc.querySelector(".issue em")).toBeTruthy();
+      expect(doc.querySelector(".req em")).toBeTruthy();
+      expect(doc.querySelector(".req h3")).toBeTruthy();
     }).then(done);
   });
 
   it("should remove left padding before processing markdown content", function(done) {
     var ops = {
       config: makeBasicConfig(),
       body: makeDefaultBody() +
-       "\n    Foo\n    ===\n",
+       "\n\n  Foo\n  ===\n    * list item 1\n    * list item 2\n      * nested list item",
```

Comment:
I screwed up the processing where I was chopping off too much off the front of strings. This test helped (me) make sure I didn't do that: hence the list inside the list. 

Tl;dr: I broke your code (which was fine); this is how I made sure I fixed it.😊

> On 5 May 2016, at 8:06 PM, Tobie Langel <notifications@github.com> wrote:
> 
> In tests/spec/core/markdown-spec.js:
> 
> >      }).then(done);
> >    });
> >  
> >    it("should remove left padding before processing markdown content", function(done) {
> >      var ops = {
> >        config: makeBasicConfig(),
> >        body: makeDefaultBody() +
> > -       "\n    Foo\n    ===\n",
> > +       "\n\n  Foo\n  ===\n    * list item 1\n    * list item 2\n      * nested list item",
> Changing the test here seems weird.
> 
> I was testing for:
> 
>     Foo
>     ===
> You're now testing for:
> 
>   Foo
>   ===
>     * list item 1
>     * list item 2
>       * nested list item
> which doesn't seem to test for the same thing.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>